x-pack/solutions/security/test/security_solution_api_integration/test_suites/lists_and_exception_lists/exception_lists_items/trial_license_complete_tier/lists/read_exception_lists.ts

```diff
@@ -135,6 +138,73 @@ export default ({ getService }: FtrProviderContext) => {
           message: 'exception list list_id: "c1e1b359-7ac1-4e96-bc81-c683c092436f" does not exist',
         });
       });
+
+      describe('@skipInServerless with read rules and all exceptions role', () => {
+        const role = ROLES.rules_read_exceptions_all;
+
+        beforeEach(async () => {
+          await createUserAndRole(getService, role);
+        });
+
+        afterEach(async () => {
+          await deleteUserAndRole(getService, role);
+        });
+        it('should be able to read a single exception list using list_id', async () => {
+          const restrictedUser = { username: 'rules_read_exceptions_all', password: 'changeme' };
+          const restrictedApis = exceptionsApi.withUser(restrictedUser);
+
+          // create a simple exception list to read
+          await supertest
+            .post(EXCEPTION_LIST_URL)
+            .set('kbn-xsrf', 'true')
+            .send(getCreateExceptionListMinimalSchemaMock())
+            .expect(200);
+
+          const { body } = await restrictedApis
+            .readExceptionList({
+              query: { list_id: getCreateExceptionListMinimalSchemaMock().list_id },
+            })
+            .expect(200);
+
+          const bodyToCompare = removeExceptionListServerGeneratedProperties(body);
+          expect(bodyToCompare).to.eql(
+            getExceptionResponseMockWithoutAutoGeneratedValues(await utils.getUsername())
+          );
+        });
+      });
+      describe('@skipInServerless with read rules and read exceptions role', () => {
```

Comment:
should we test that API returns 403 when no exceptions sub-feature enabled?
Similar would go to the rest of read APIs